lib/execGit.js

```diff
@@ -1,18 +1,19 @@
-'use strict'
+import debug from 'debug'
+import execa from 'execa'
 
-const debug = require('debug')('lint-staged:git')
-const execa = require('execa')
+const debugLog = debug('lint-staged:execGit')
```

Comment:
I made sure the debug logger is always called `debugLog`, and its scope is the actual method name.